### PR TITLE
Jcs/reject reason

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

**Comment by jSylvestre** [comment]
resolve #61

**Comment by jSylvestre** [comment]
image

File: Hippo.Web/ClientApp/src/Shared/RejectRequest.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          <textarea
+            className="form-control"
+            id="fieldName"
+            rows={3}
+            required
+            value={reason}
+            onChange={(e) => {
+              setReason(e.target.value);
+              setReturn(e.target.value);
+            }}
```
Comment (sprucely) [inline Hippo.Web/ClientApp/src/Shared/RejectRequest.tsx:37]: Should we do something here to enforce the 1500 char limit? Or should we wait until we have yup validation in place?

**Review by sprucely — APPROVED** [review]

File: Hippo.Web/ClientApp/src/Shared/RejectRequest.tsx (hunk 1 above)
Comment (jSylvestre) [reply Hippo.Web/ClientApp/src/Shared/RejectRequest.tsx:37]: That was me just pulling a number out of my hat... Email would still get the full text

**Pull request merged**